lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -22,6 +22,7 @@ class LogEventAggregator < EventAggregator
       METRICS_SUPPORTABILITY_FORMAT = "Supportability/Logging/Metrics/Ruby/%s".freeze
       FORWARDING_SUPPORTABILITY_FORMAT = "Supportability/Logging/Forwarding/Ruby/%s".freeze
       DECORATING_SUPPORTABILITY_FORMAT = "Supportability/Logging/LocalDecorating/Ruby/%s".freeze
+      MAX_BYTES = 32768 # 32 * 1024 = 1 kibibyte
```

Comment:
The way this comment is written might confuse some.

This part is helpful:

```ruby
32768 # 32 * 1024
```

This part is helpful:

```ruby
1024 = 1 kibibyte
```

But when both of these parts are combined, it looks like we're saying that 32,768 bytes equals 1 kibibyte (whereas it's really 32 kibibytes).

Reply:
That's my bad, I forgot that the limit was 32 kb. Would you like me to change the text to show 32 kibibytes instead of 1 kibibyte?

Reply:
Yeah, I think that would read better. Something like this:

```ruby
MAX_BYTES = 32768 # 32 * 1024 bytes (32 kibibytes)
```